CHANGELOG.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+**Breaking API**
```

Comment:
Just put this under a Forthcoming header (with underline), no need to mark them as Breaking/New (they're not actually breaking API changes even since the old meta api is still there.

Reply:
I'll make an OSRC build farm release as v0.5.11 once you've worked through this with any fixes that will follow.

Reply:
Committed.